app/components/CacheBuster/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+
+const CacheBuster = ({
+  children,
+}) => {
+  useEffect(() => {
+    const delCaches = async () => {
+      const cacheKeys = await caches.keys();
```

Comment:
caches is undefined in development mode -- think we can add a check here for `window.caches` ?

Reply:
Is it showing an error in the console? I'm not seeing anything

Reply:
weird, probably because I'm not under SSL, are you? if you are then caches won't be undefined, you think that'd be a problem?

Reply:
I'm not on SSL, no. I looked it up though and you're right so I'll add some logic for if it's undefined